etna/core/models/basepage.py

```diff
@@ -52,6 +53,8 @@ class BasePage(MetadataPageMixin, DataLayerMixin, Page):
         help_text=_("Image that will appear on thumbnails and promos around the site."),
     )
 
+    uuid = models.UUIDField("UUID", unique=True, default=uuid4, editable=False)
```

Comment:
Hi @TNA-Allan. I simply thought it would be nicer to generate UUIDs from existing page titles, since we have them to hand. However, we can't do the same thing via the `default` argument on the field, because that isn't 'instance specific' code. To do the same thing for new pages, we'd need to override `full_clean()` (and possibly also `save()`) to get access to other field values, so it didn't feel worthwhile.